app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadAdapter.kt

```diff
@@ -69,19 +69,21 @@ class ThreadAdapter(
     private val isForPrinting: Boolean = false,
     private val isCalendarEventExpandedMap: MutableMap<String, Boolean> = mutableMapOf(),
     private var threadAdapterCallbacks: ThreadAdapterCallbacks? = null,
-) : ListAdapter<Message, ThreadViewHolder>(MessageDiffCallback()) {
+) : ListAdapter<Message, MessageViewHolder>(MessageDiffCallback()) {
 
     inline val messages: MutableList<Message> get() = currentList
 
     var isExpandedMap = mutableMapOf<String, Boolean>()
+
+    //region Auto-scroll at Thread opening
     var initialSetOfExpandedMessagesUids = setOf<String>()
     private val currentSetOfLoadedExpandedMessagesUids = mutableSetOf<String>()
     private var hasNotScrolledYet = true
+    //endregion
```

Comment:
Maybe we can also group `public` and `private` variables?

Reply:
We sort them by logic.